includes/challenges-13.test.js

```diff
@@ -0,0 +1,273 @@
+
+const allAnything = (strs, target) => {
+  return strs.every( string => string.includes(target)); 
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 8
+//
+// We've been testing a new course enrollment system, and we think we have the bugs worked out, but
+// in the meantime, Brook enrolled himself in a bunch of different classes to test if it was working.
+//
+// Write a function named unenrollBrook that takes in a 2d array where each array represents
+// one course's roster, and is an array of strings of the names of the people in that course.
+// Return a 2d array with the same roster, but where anyone whose name includes Brook
+// is removed from every course.
+// ------------------------------------------------------------------------------------------------
+
+const unenrollBrook = (roster) => {
+  return roster.map(arr => arr.filter( string => !string.includes('Brook')));
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 9
+//
+// Write a function named sortByDay that takes in an array of strings, each of which represents an
+// event's day and time.
+//
+// Return a 2d array that organizes those strings based on the day they happen.
+// For example, all Monday's events are in the first array, all Tuesday's are in the second, etc.
+//
+// If an event takes place on multiple days (i.e. "Dancing on Mondays and Tuesdays"), it should
+// appear in both arrays.
+// ------------------------------------------------------------------------------------------------
+
+const daysOfWeek = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+
+const sortByDay = (eventStrings) => {
+  return eventStrings.reduce( (arr, string) => {
+    if(string.includes('Monday')){
+      arr[0].push(string);
+    }
+    if(string.includes('Tuesday')){
+      arr[1].push(string);
+    }
+    if(string.includes('Wednesday')){
+      arr[2].push(string);
+    }
+    if(string.includes('Thursday')){
+      arr[3].push(string);
+    }
+    if(string.includes('Friday')){
+      arr[4].push(string);
+    }
+    if(string.includes('Saturday')){
+      arr[5].push(string);
+    }
+    if(string.includes('Sunday')){
+      arr[6].push(string);
+    }
+    return arr;
+  },[[],[],[],[],[],[],[]]);
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 10
+//
+// Write a function named characterByIndex that takes in an array of strings and returns an array containing
+// the first character of the first string, the second character of the second string, etc.
+// ------------------------------------------------------------------------------------------------
+
+const characterByIndex = (strs) => {
+  const newArr = [];
+  strs.forEach( (string, idx) => {
+    newArr.push(string.charAt(idx));
+  });
+  return newArr;
+};
```

Comment:
you pretty much wrote a map() function, `return strs.map((str, idx) => str.charAt(idx));`